lib/core/src/lib/card-view/components/card-view-dateitem/card-view-dateitem.component.scss

```diff
@@ -11,33 +26,24 @@
         float: right;
     }
 
-    &-dateitem-chip-list-container.adf-property-field {
+    .adf-dateitem-chip-list-container.adf-property-field {
         margin-bottom: -7px !important;
         border-bottom: 0;
         cursor: pointer;
-
-        .adf-dateitem-editable-controls {
-            margin-top: 15px;
-        }
-
-        .mat-datetimepicker-toggle {
-            position: absolute;
-            right: 0;
-            top: -20px;
-        }
     }
 
-    &-dateitem-editable {
+    .adf-dateitem-editable {
         cursor: pointer;
-        border-bottom: 1px solid var(--adf-theme-foreground-text-color-042);
```

Comment:
I can't see usages of this variable --adf-theme-foreground-text-color-042 so probably it can be removed now.

Reply:
Done